index.html

```diff
@@ -0,0 +1,785 @@
+            <ul class="nav-ul">
+              <li><a href="#">About</a></li>
+              <li><a href="#">Advertise</a></li>
+              <li><a href="#">Jobs</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </div>
+        </nav>
+        <nav class="na-3">
+          <ul class="nav-ul">
+            <li><a href="#">LATEST</a></li>
+            <li><a href="#">HARD FORK</a></li>
+            <li><a href="#">PLUGGED</a></li>
+            <li><a href="#">README</a></li>
+            <li><a href="#">GROWTH QUARTERS</a></li>
+            <li><a href="#">SHIFT</a></li>
+            <li><a href="#">NEURAL</a></li>
+          </ul>
+          <img
+            class="srchicon"
+            alt="search icon"
+            src="Assets/Images/magnifying-glass.svg"
+          />
+        </nav>
+        <nav class="na-4">
+          <a href="#"
+            >Calling young digital talent in the Netherlands – Be recognized in
+            the T500 →</a
+          >
+        </nav>
+      </nav>
+    </section>
+
+    <section class="main-sec">
+      <!--main tag-->
+      <section class="article3">
+        <article class="art-cont art-main">
+          <div class="art1 art123">
+            <a class="art-cover" href="#"></a>
+            <header class="arthead">
+              <div class="big-logo">
+                <img alt="" src="Assets/Images/big-logo.svg">
+              </div>  
+              <div class="date">Tuesday — January 26, 2021</div>
+            <div class="opal">
+                Opal cuts off your apps' internet access to help you stop
+                doomscrolling
+            </div>
+              <div>
+                <ul>
+                  <li>by ivan Mehta</li>
+                  <li>1 hour ago</li>
+                  <li>6</li>
+                </ul>
+              </div>
+            </header>
+          </div>
+        </article>
+        <article class="art-cont art-tr">
+          <div class="art2 art123">
+            <a class="art-cover" href="#"></a>
+            <header class="arthead">
+              <h2>Google unveils privacy-friendly AI cookie killer</h2>
+              <div>
+                <ul>
+                  <li>by Thomas Macauly</li>
+                  <li>2 hour ago</li>
+                  <li>4</li>
+                </ul>
+              </div>
+            </header>
+          </div>
+        </article>
+        <article class="art-cont art-br">
+          <div class="art3 art123">
+            <a class="art-cover" href="#"></a>
+            <header class="arthead">
+              <h2>This Mac app automatically detects your duplicate files</h2>
+              <div>
+                <ul>
+                  <li>by ivan Mehta</li>
+                  <li>2 hour ago</li>
+                  <li>5</li>
+                </ul>
+              </div>
+            </header>
+          </div>
+        </article>
+      </section>
+      
+    <!--<div id="title1">
```

Comment:
## Remove old pieces of commented code.
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history just as explained [here](https://microverse.zendesk.com/hc/en-us/articles/360057489093-General-Requirements-for-Microverse-Projects).

Reply:
As requested previously, remove all commented out code from this file.